src/test/java/Homework19.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Homework19 extends BaseTest{
+
+    @Test(enabled = true)
+    public void testDeletePlaylist()throws InterruptedException{
+        //
+        String namePl   = "MyPlaylist";
+        String selector = "//section[@id='playlists']//li//a[contains(text(),'"+namePl+"')]";
+
+        login();
+
+        try{
+            WebElement MyPlaylist = findElement("//section[@id='playlists']//li//a[contains(text(),'"+namePl+"')]");
+        }
+        catch (Exception e) {
+            createPlaylist(namePl);
+        }
+
+        WebElement PL = findElement(selector);
```

Comment:
This variable should be renamed to something more descriptive , like playlist

**I recommend to use in the variable naming:**
-Follow code convention
-Use comprehensive names
-1 and 0 harcoded


Java convention:
- i and j allowed for loops
- ALL_CAPS - constant
-  ClassFromCapital
-  methodOrVariable

AVOID
•	Magic numbers
•	Hungarian notation L
•	Abbreviations
•	x, y, z or any one-letter apart i/j
•	file2, sum1
•	Unclear "flag" "result" "status"

Reply:
ok, variables was renamed